Extend/Controller/OrderController.php

```diff
@@ -369,13 +371,52 @@ private function _getCreditCardFormRequestData()
 
         $oPayment = PaymentMethodHelper::getPaymentById($oBasket->getPaymentId());
         $sPaymentAction = $this->_getPaymentAction($oPayment->oxpayments__wdoxidee_transactionaction->value);
-        $sLanguageCode = Registry::getLang()->getLanguageAbbr();
+        $sLanguageCode = $this->_getSupportedLanguageCode();
 
         $oTransactionService = $this->_getTransactionService();
 
         return $oTransactionService->getCreditCardUiWithData($oTransaction, $sPaymentAction, $sLanguageCode);
     }
 
+    /**
+     * Gets the supported WPP language code
+     *
+     * @return string
+     *
+     * @since 1.3.0
+     */
+    private function _getSupportedLanguageCode()
+    {
+        $wConverter = new WppVTwoConverter();
+        try {
+            $wConverter->init();
+            $wLanguage = $wConverter->convert(
+                $this->_removeSuffix(Registry::getLang()->getLanguageAbbr())
+            );
+        } catch (\Exception $wException) {
+            $wLanguage = self::DEFAULT_WPP_LANGUAGE;
+        }
+
+        return $wLanguage;
+    }
+
+    /**
+     * @param string $oLangCode
+     * @param string $wCutOffPoint
+     *
+     * @return string
+     *
+     * @since 1.3.0
+     */
+    private function _removeSuffix($oLangCode, $wCutOffPoint = '_')
```

Comment:
@sebastijansakac just a question. why are you adding the cutOffPoint as a second param to the function if its not called with this param and then you can just use the '_' directly inside?

Reply:
It was copy/paste from PS, as mentioned in the ticket. I just adjusted for oxid